files/workflows/ZWEWRF05.xml

```diff
@@ -0,0 +1,215 @@
+        <category>General security</category>
+        <string valueMustBeChoice="false" multiLine="false">
+            <default>ZWEADMIN</default>
+        </string>
+    </variable>
+    <step name="define_variables" optional="false">
+		<title>Define Variables</title>
+		<description>Step to define the variables</description>
+        <variableValue name="zowe_install_path" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="USSDIR" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="HOSTNAME" scope="instance" noPromptIfSet="false" required="false"/>
+        <variableValue name="IPADDRESS" scope="instance" noPromptIfSet="false" required="false"/>
+        <variableValue name="VERIFY_CERTIFICATES" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="EXTERNAL_CERTIFICATE" scope="instance" noPromptIfSet="false" required="false"/>
+        <variableValue name="EXTERNAL_CERTIFICATE_ALIAS" scope="instance" noPromptIfSet="false" required="false"/>
+        <variableValue name="EXTERNAL_CERTIFICATE_AUTHORITIES" scope="instance" noPromptIfSet="false" required="false"/>
+        <variableValue name="ZOSMF_CERTIFICATE" scope="instance" noPromptIfSet="false" required="false"/>
+        <variableValue name="KEYSTORE_PASSWORD" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="KEYSTORE_DIRECTORY" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="KEYSTORE_ALIAS" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="ZOWE_USER_ID" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="ZOWE_GROUP_ID" scope="instance" noPromptIfSet="false" required="true"/>
+		<instructions substitution="false">Run this step to initialize variable values</instructions>
+		<weight>1</weight>
+		<autoEnable>false</autoEnable>
+		<canMarkAsFailed>false</canMarkAsFailed>
+	</step>
+    <step name="Substitute_vars_in_env_file" optional="true">
+        <title>New Custom zowe-setup-certificates.env</title>
+        <description>Creates new zowe-setup-certificates.env in user specified location adn substitutes values</description>
+        <prereqStep name="define_variables"/>
+        <instructions substitution="false">This step creates zowe-setup-certificates.env and substitutes variables</instructions>
+        <weight>1</weight>
+        <skills>z/OS System Programmer</skills>
+        <autoEnable>true</autoEnable>
+        <canMarkAsFailed>true</canMarkAsFailed>
+        <template>
+            <inlineTemplate substitution="true">
+if test ! -d ${USSDIR}; then mkdir -p -m 755 ${USSDIR}; fi
+cp ${zowe_install_path}/bin/zowe-setup-certificates.env ${USSDIR}/zowe-setup-certificates.env
+chmod 755 ${USSDIR}/zowe-setup-certificates.env
+
+HOSTNAME='${HOSTNAME}'
```

Comment:
Since we are not generating the workflow and zowe-setup-certificates.env from a single source, we should add a comment to zowe-setup-certificates.env indicating that development must keep it in sync with this workflow, and vice versa. Other option of course is to rework the setup in Github so we can single source the two files that end up being shipped.

Reply:
@OnnoVdT I missed that comment on top, sorry.
I don't think anyone will want to change the workflow without editing the zowe-setup-certificates.env file, since the wf is covering the variables in the zowe-setup-certificates.env. Maybe for now we can just add instructions in the zowe-setup-certificates.env file and in the future think of a way to generate them both from a single source?

Reply:
I agree, hence that being my first proposition. A note in the .env should be sufficient as any changes to the process will likely be tested without the workflow in play, and thus the comment will be noticed.